impeller/compiler/shader_lib/impeller/gaussian.glsl

```diff
@@ -9,16 +9,16 @@
 
 /// Gaussian distribution function.
 float IPGaussian(float x, float sigma) {
-  float variance = sigma * sigma;
-  return exp(-0.5 * x * x / variance) / (kSqrtTwoPi * sigma);
+  float variance = pow(sigma, 2.0);
```

Comment:
These changes seem to give a minor improvement to register usage but also seeminly reduce the % of 16bit computation since the shaders produced by impellerc seem to always convert mediump to highp.

Reply:
Overall, marginal win - could be bigger if we didn't lose mediump tho